_dev/src/components/product-feed/settings/delivery-time-and-rates/delivery-time-and-rates.vue

```diff
@@ -123,31 +111,62 @@ export default Vue.extend({
       });
       window.scrollTo(0, 0);
     },
-    saveSelectedCountries() {
+    validateForm(): boolean {
+      this.displayValidationErrors = true;
+
+      // Validation - Target countries
+      if (!this.countries.length) {
+        return false;
+      }
+
+      // Validation - Estimate option
+      if (this.getShippingValueSetup === ShippingSetupOption.ESTIMATE) {
+        if (validateCarrier(this.customCarrier) === false) {
+          return false;
+        }
+        return true;
+      }
+
+      // Validation - Import option
+      if (this.getShippingValueSetup === ShippingSetupOption.IMPORT) {
+        const enabledDeliveryDetails: DeliveryDetail[] = this.carriers.filter(
+          (e: DeliveryDetail) => e.enabledCarrier,
+        );
+
+        if (!enabledDeliveryDetails.length) {
+          return false;
+        }
+
+        return enabledDeliveryDetails.every(validateDeliveryDetail);
+      }
+
+      return false;
+    },
+    saveSelectedCountries(): void {
       localStorage.setItem('productFeed-targetCountries', JSON.stringify(this.countries));
       this.$store.commit('productFeed/SET_SELECTED_PRODUCT_FEED_SETTINGS', {
         name: 'targetCountries', data: this.countries,
       });
     },
-    nextStep() {
-      if (validateCarrier(this.customCarrier) === false) {
-        this.validationError = true;
-        return;
-      }
-      this.validationError = false;
-
-      if (this.getShippingValueSetup === ShippingSetupOption.IMPORT) {
```

Comment:
@PoulainMaxime This seemed to be wrong, can you confirm?

Reply:
Yes it's wrong, I confused Import and Estimate here